docs/integrations/databases/opentelemetry/memcached-opentelemetry.md

```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
 | timeslice by 1h 
 | sum(ERROR) as ERROR by _timeslice
 ```
+## Sample metrics query
```

Comment:
These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample metrics queries
```